docker_settings.yaml

```diff
@@ -11,6 +11,7 @@ admin:
 
 addresses:
   oa:
+    nb_threads: 4
```

Comment:
Could you add a default (empty) value in `invoke.yaml` as well, so that all settings are documented in one place ?